CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.0.0-rc.4
+
+**Release date:** 2023-05-26
+
+This release candidate comes with support for Kubernetes v1.27 and Cosign v2.
+It also enables the use of annotated tags with `spec.ref.name`. Furthermore, it
```

Comment:
Annotated tags in what object kind? Also: `.spec.ref.name`, for full YAML dot notation path.